public/app/plugins/datasource/prometheus/components/PrometheusMetricsBrowser.tsx

```diff
@@ -158,6 +158,9 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => ({
   `,
   valueList: css`
     margin-right: ${theme.spacing.sm};
+    min-width: 300px;
+    max-width: 900px;
```

Comment:
`max-width` shouldn't be set to random number, because in many cases, this is not going to work (explore split view, query builder in dashboard). In this case, `max-width: 100%` should work, but unfortunately it does not. This is also an issue in https://github.com/grafana/grafana/issues/52301, so I am going to create separate issue to ensure, that we can use `max-width:100%` `width:100%`. Therefore I'd say to remove `max-width` and we solve in in separate PR.